util/include/Digest.hpp

```diff
@@ -24,10 +21,13 @@ using BlockDigest = std::array<std::uint8_t, DIGEST_SIZE>;
 
 #if defined USE_CRYPTOPP_HASH
 using Digest = DigestHolder<CryptoppDigestCreator>;
+using DigestUtil = CryptoppDigestCreator;
```

Comment:
Can we remane DigestUtil to Digester? It will be a synonym of Digest Creator.

Reply:
Addressed https://github.com/arc-vmware/concord-bft/pull/3/commits/93e565e22eae96ba5c40756c974a391f8bcb282d